server/document-watch.worker.js

```diff
@@ -14,6 +14,14 @@ function postEvent(type, data = {}) {
 }
 
 function postDocumentInfo(filePath, changeType) {
+  // Note! On Windows, `glob.sync()` will return URLs that always use
+  // the `/` character (POSIX STYLE, https://www.npmjs.com/package/glob#windows)
+  // But the `Document.urlToFolderPath()` function will respect Windows
+  // notation and use `\` characters.
+  // Now, when we use it, it'll work as if `glob.sync()` and returned paths
+  // in a fashion that is expected in the OS.
+  filePath = filePath.split("/").join(path.sep);
```

Comment:
I would move this to line 56 https://github.com/mdn/yari/pull/2748/files#diff-7c825b83e0a9311cc1fe49ed6075267dbcd2c0aad8c61a008d178909d2633f07R56